### ISSUE TITLE
Small fix for IPv6 multicast support

### ISSUE THREAD

**Issue by darkopetrovic** [contiki-os/contiki#1292]
In the `uip_ds6_select_src()` function, `UIP_IPV6_MULTICAST` shouldn't it be `UIP_CONF_IPV6_MULTICAST`?  `UIP_IPV6_MULTICAST` doesn't exist. 

And then add `#include "net/ipv6/multicast/uip-mcast6.h"`.

**Comment by g-oikonomou** [comment]
Yes you are right, this looks like a refactor gone wrong. This also needs changed:

```
#endif /* UIP_IPV6_MULTICAST */
```

https://github.com/contiki-os/contiki/blob/master/core/net/ip/uip-udp-packet.c#L67

Since this define is not a user configuration, I'd prefer using `UIP_IPV6_MULTICAST` across the board
